molecule/default/prepare.yml

```diff
@@ -0,0 +1,33 @@
+- name: Install nginx
+  hosts: all
+  # connection: local
+  become: no
+  gather_facts: yes
+  tasks:
+
+    - name: Update yum
+      yum:
+        update_cache: yes
+      when: ansible_os_family == 'RedHat'
+
+    - name: Ensure nginx is installed
+      package:
+        name:
+          - nginx
+        state: present
+      when: ansible_os_family == 'RedHat'
+
+    - name: Ensure python is installed
+      package:
+        name:
+          - python
```

Comment:
@Chithrak07 You might need python3 here? Python2 is Dead/EOL. This is just a guess. You'll need to find out for sure what the package name is for RH/CentOS.